gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
-|  wheelchair_boarding | Enum | Optional | The **wheelchair_boarding field** identifies whether wheelchair boardings are possible from the specified stop, station, or station entrance. The field can have the following values: |
-|   |  | | * **0** (or empty) - indicates that there is no accessibility information for the stop |
-|   |  | | * **1** - indicates that at least some vehicles at this stop can be boarded by a rider in a wheelchair | 
-|   |  | | * **2** - wheelchair boarding is not possible at this stop |  
-|   |  | | When a stop is part of a larger station complex, as indicated by a stop with a **parent_station** value, the stop's **wheelchair_boarding** field has the following additional semantics: |  
-|   |  | | * **0** (or empty) - the stop will inherit its **wheelchair_boarding** value from the parent station, if specified in the parent |  
-|   |  | | * **1** - there exists some accessible path from outside the station to the specific stop / platform | 
-|   |  | | * **2** - there exists no accessible path from outside the station to the specific stop / platform | 
-|   |  | | For station entrances, the **wheelchair_boarding** field has the following additional semantics: | 
-|   |  | | * **0** (or empty) - the station entrance will inherit its **wheelchair_boarding** value from the parent station, if specified in the parent |  
-|   |  | | * **1** - the station entrance is wheelchair accessible (e.g. an elevator is available to platforms if they are not at-grade)  |  
-|   |  | | * **2** - there exists no accessible path from the entrance to station platforms |  
+|  stop_id | ID | **Required** | Identifies a stop, station, or station entrance/exit. Stops, stations or station entrances/exits are collectively referred to as locations. Multiple routes may use the same stop. | 
+|  stop_code | Text | Optional | Short text or a number that identifies the location for riders. These codes are often used in phone-based transit information systems or printed on signage to make it easier for riders to get information for a particular location. The `stop_code` can be the same as `stop_id` if it is public facing. This field should be left "empty" for locations without a code presented to riders. | 
+|  stop_name | Text | **Required** | Name of a location. Use a name that people will understand in the local and tourist vernacular. | 
```

Comment:
Above you said:

> ... identifies *the* location

And here you say

> ... of *a* location

We likely should always use the same article.

IMHO "the" make sense, since they are properties of an object.